src/PortToDocs/src/libraries/ToDocsPorter.cs

```diff
@@ -559,34 +628,86 @@ private void TryPortMissingParamsForAPI(IDocsAPI dApiToUpdate, IntelliSenseXmlMe
         // Ports all the type parameter descriptions for the specified API if any of them is undocumented.
         private void TryPortMissingTypeParamsForAPI(IDocsAPI dApiToUpdate, IntelliSenseXmlMember? tsMemberToPort, DocsMember? interfacedMember)
```

Comment:
This whole method is identical to `TryPortMissingeParamsForAPI`, but specific to type parameters. It prompts the user like we do with parameters, if the names do not match.